index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>David's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+  <div class ="container">
+    <header class>
+      <a href="#"class="logo">My Logo</a>
+      <nav>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </nav>
+    </header>
+
+    <section class = "section-1">
+      <h1> I'm David <br>Glad to see <br>you!</h1>
+      <p> I’m a software developer! I can help you build a product,
+        feature or website Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#"> LET'S CONNECT</a>
+      <ul class="social-image">
+        <li><a href="#"><img src="images/twitter_1.png" alt="Twiiter logo"></li>
+        <li><a href="#"><img src="images/Linkedin_1.png" alt="in logo"></li>
+        <li><a href="#"><img src="images/medium.png" alt="m logo"></li>
+        <li><a href="#"><img src="images/github1.png" alt="github logo"></li>
+        <li><a href="#"><img src="images/angellist.png" alt="angel logo"></li>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to add a link to each of your social media pages, for example, Twitter, LinkedIn, and GitHub, ... to help recruiters to connect with you online. 
Kindly add links to your social media icons in corresponding `<a>` anchor tags.
for example : `<a href="https://github.com/david30-maker">`